packages/ui/src/components/LegendItem/index.tsx

```diff
@@ -42,6 +44,11 @@ function LegendItem(props: Props) {
                     className={_cs(styles.color, colorClassName)}
                 />
             )}
+            {icons && (
+                <div>
+                    {icons}
+                </div>
+            )}
```

Comment:
We already have iconSrc. We don't need icons prop

Reply:
iconSrc is for image path, icons prop is the for the icons, types are different